packages/theme/api-report.md

```diff
@@ -72,6 +72,17 @@ export type BackstagePaletteAdditions = {
     link: string;
     warning?: string;
   };
+  catalogGraph?: {
+    component?: string;
+    domain?: string;
+    system?: string;
+    location?: string;
+    resource?: string;
+    group?: string;
+    user?: string;
+    template?: string;
+    api?: string;
+  };
```

Comment:
Would very much like to avoid this tbh. The theme is supposed to be as generic as possible, so adding things for individual plugins is something we want to avoid. Is there any other API surface you see that might make sense for this customization?

Reply:
@Rugvip I agree with you, I implemented it in theme because any person who wanted this feature will have to create their own themes and add `catalogGraph` property to it. 
At this moment I have no idea, Do you have any suggestions for where this customization can be added?

Reply:
Tbh this is a bit of a gap atm, I think this is a very good use-case for #11404 though so I'm feeling we want to wait for that